### ISSUE TITLE
Typescript doesn't find .obj files

### ISSUE THREAD

**Issue by domhel** [tonyketcham/unplugin-obj#1]
I'm using SvelteKit and Typescript.
When trying to import and .obj file like this
```ts
	// eslint-disable-next-line @typescript-eslint/ban-ts-comment
	// @ts-ignore
	import tree from '$/lib/components/models/PalmTree_1.obj'
```
I get this error in VSCode
`Cannot find module '../lib/assets/PalmTree_1.obj' or its corresponding type declarations. ts(2307)`
and this error
```cmd
1:15:43 PM [vite] Error when evaluating SSR module /src/routes/index.svelte:
Error: failed to load module for ssr: /src/lib/components/models/PalmTree_1.obj
    at instantiateModule (/home/dominik/dev/cubed-test/node_modules/vite/dist/node/chunks/dep-7e125991.js:67246:15)
failed to load module for ssr: /src/lib/components/models/PalmTree_1.obj
Error: failed to load module for ssr: /src/lib/components/models/PalmTree_1.obj
    at instantiateModule (/home/dominik/dev/cubed-test/node_modules/vite/dist/node/chunks/dep-7e125991.js:67246:15)
```
in the SvelteKit console.

I've tried it with many different paths, files and the paths are definitely correct.

I'm using this in a brand new svelte@next environment with svelte-cubed and three.

I haven't tried it in JavaScript yet, maybe it's working in JS?

Edit:
My Svelte config looks like this
```ts
import adapter from '@sveltejs/adapter-auto';
import preprocess from 'svelte-preprocess';
import ObjFileImport from 'unplugin-obj/vite';


/** @type {import('@sveltejs/kit').Config} */
const config = {
	// Consult https://github.com/sveltejs/svelte-preprocess
	// for more information about preprocessors
	preprocess: preprocess(),

	kit: {
		adapter: adapter(),

		// hydrate the <div id="svelte"> element in src/app.html
		target: '#svelte',
		vite: {
			plugins: [
				ObjFileImport()
			]
		}
	}
};

export default config;
```

**Comment by tonyketcham** [comment]
Yeah, Rollup was making the typing of those `*.obj` modules complicated since it has shoddy module declaration support in plugins. I came up with a solution that allows the bundler to opt-in to typings, so it gets around the Rollup issue and allows Vite (for example) to take advantage of the types.

I just released an update `unplugin-obj@0.1.2` with a [SvelteKit/svelte-cubed example](https://github.com/tonyketcham/unplugin-obj/tree/master/playground-sveltekit) to follow which should fix this for ya!
